dist/main.js

```diff
@@ -0,0 +1,14 @@
+
+const expenseManager = new ExpenseManager()
+const render = new Renderer()
+
+const run = async function(){
+    await expenseManager.getExpenses()
+    console.log(expenseManager.expenses)
+    render.renderExpenses(expenseManager.expenses)
+}
+
+run()
```

Comment:
if you have a HTML file, you could have the `run` function work with a click of a button...